pkg/render/manager.go

```diff
@@ -92,14 +92,8 @@ func Manager(
 	copy.ObjectMeta = metav1.ObjectMeta{Name: ManagerTLSSecretName, Namespace: ManagerNamespace}
 	tlsSecrets = append(tlsSecrets, copy)
 
-	var tunnelSecrets []*corev1.Secret
+	var tunnelSecrets = []*corev1.Secret{tunnelSecret}
 	if management {
-		// If there is no secret create one and add it to the operator namespace.
-		if tunnelSecret == nil {
-			tunnelSecret = voltronTunnelSecret()
-			tunnelSecrets = append(tunnelSecrets, tunnelSecret)
-		}
-
 		tunnelSecrets = append(tunnelSecrets, CopySecrets(ManagerNamespace, tunnelSecret)...)
```

Comment:
I guess our approach here is that we won't have the operator call this out to the user. 

The new normal is that the user must always provide a tunnel cert (just like they must provide a tunnel service) manually. 

So we just need to document it in our user docs. 

I recall this is what was discussed already. I'm just writing it out just to confirm and job my own memory.

Reply:
In reference to your 1st sentence. The operator will report a missing certificate if one is not provided. See above at https://github.com/tigera/operator/pull/509/files#diff-8962c60cf9c10aa1ac7a2ffb1422ae3cR303